src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -18762,6 +18762,58 @@ public void PrimaryConstructors_17([CombinatorialValues("class", "struct")] stri
                 Diagnostic(RudeEditKind.Update, "long a", FeaturesResources.parameter));
         }
 
+        [Theory]
+        [CombinatorialData]
+        public void PrimaryConstructors_18([CombinatorialValues("class", "struct")] string keyword)
```

Comment:
Updates in field initializers need to produce an update to the constructory symbol, not the field symbol. Field symbol updates should only be produced when the field itself changes (e.g. an attribute applied to the field changes).

See e.g. `FieldInitializerUpdate_Lambdas_ImplicitCtor_EditInitializerWithLambda1` test.

Reply:
> Updates in field initializers need to produce an update to the constructory symbol, not the field symbol. Field symbol updates should only be produced when the field itself changes (e.g. an attribute applied to the field changes).

There is no goal to provide perfect EnC experience for the tested scenarios. I am comfortable with any error as long as it is a rude edit error.